the-blue-alliance-ios/View Elements/Events/RankingTableViewCell.swift

```diff
@@ -11,18 +11,36 @@ import UIKit
 
 class RankingTableViewCell: UITableViewCell {
     static let reuseIdentifier = "RankingCell"
-    public var ranking: DistrictRanking? {
+    public var eventRanking: EventRanking? {
         didSet {
-            guard let ranking = ranking, let team = ranking.team else {
+            guard let ranking = eventRanking, let team = ranking.team else {
+                return
+            }
+            rankLabel?.text = "Rank \(ranking.rank)"
+            numberLabel?.text = "\(team.teamNumber)"
+            nameLabel?.text = team.nickname ?? team.fallbackNickname
+            if let qualAverage = ranking.qualAverage as? Double {
+                detailLabel?.text = "Avg. \(qualAverage) Points"
+            }
+            self.setupWLTLabel(ranking: ranking)
+        }
+        
+    }
+    
+    public var districtRanking: DistrictRanking? {
+        didSet {
+            guard let ranking = districtRanking, let team = ranking.team else {
                 return
             }
             rankLabel?.text = "Rank \(ranking.rank)"
             numberLabel?.text = "\(team.teamNumber)"
             nameLabel?.text = team.nickname ?? team.fallbackNickname
             detailLabel?.text = "\(ranking.pointTotal) Points"
+
```

Comment:
done. I think I'm starting to get the hang of this project's syntax standards. Sorry it's taken so long.